android/app/src/main/java/org/uwaterloo/subletr/navigation/MainNavigation.kt

```diff
@@ -69,5 +70,13 @@ fun MainNavigation(
 				modifier = modifier,
 			)
 		}
+		composable(
+			route = NavigationDestination.LISTING_DETAILS.fullNavPath,
+			arguments = listOf(navArgument("listingId") { type = NavType.StringType }),
```

Comment:
Wait, sorry, isn't this an int?

Reply:
yeah I changed it
the id being `-1908663237` made me think it had to be a string